boot.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/deis/registry-token-refresher/pkg"
+	"github.com/deis/registry-token-refresher/pkg/credentials"
+	"k8s.io/kubernetes/pkg/api"
+	kcl "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+var appList []string
```

Comment:
then, can it be local to that goroutine? I'm asking because the minute this gets accessed anywhere else, it's immediately a race